src/moments.jl

```diff
@@ -363,3 +363,44 @@ end
 
 kurtosis(v::RealArray) = kurtosis(v, mean(v))
 kurtosis(v::RealArray, wv::AbstractWeights) = kurtosis(v, wv, mean(v, wv))
+
+"""
+    cumulant(v, k, [wv::AbstractWeights], m=mean(v))
```

Comment:
Should we allow passing `corrected` here too? This should probably be mentioned anyway.

Reply:
Can you please elaborate, as moment function does not take this argument so how will it be used?

Reply:
You can check whether `wv isa UnitWeights` and if so just call `moment` without weights.

Reply:
Sorry, I didn't address the question. I thought `moment` took `corrected` like `var`, but that's not the case so everything is fine.